diff/print.go

```diff
@@ -58,11 +56,15 @@ func PrintFileDiff(d *FileDiff) ([]byte, error) {
 	return buf.Bytes(), nil
 }
 
-func timePtr(ts *pbtypes.Timestamp) *time.Time {
+func timePtr(ts []byte) *time.Time {
 	if ts == nil {
 		return nil
 	}
-	t := ts.Time()
+	var t time.Time
+	err := t.UnmarshalBinary(ts)
+	if err != nil {
+		return nil
```

Comment:
We're now ignoring an error here. Maybe we want to handle that? It seems we can just ignore it and the timestamp will be left out of the printing, but then I'd at least add a comment stating that we intend to ignore the error. Up to what the maintainers think.